appsTransport.go

```diff
@@ -0,0 +1,80 @@
+package ghinstallation
+
+import (
+	"crypto/rsa"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"time"
+
+	jwt "github.com/dgrijalva/jwt-go"
+)
+
+// AppsTransport provides a http.RoundTripper by wrapping an existing
+// http.RoundTripper and provides GitHub Apps authentication as a
+// GitHub App.
+//
+// Client can also be overwritten, and is useful to change to one which
+// provides retry logic if you do experience retryable errors.
+//
+// See https://developer.github.com/apps/building-integrations/setting-up-and-registering-github-apps/about-authentication-options-for-github-apps/
+type AppsTransport struct {
+	BaseURL       string            // baseURL is the scheme and host for GitHub API, defaults to https://api.github.com
```

Comment:
Minor issue, comment incorrectly refers to the field as `baseURL` but it's actually named `BaseURL`.

Reply:
Thanks @shurcooL and yes it is, I've fixed in both places in 3f4e9b1898e6b65e31827d5dccb97d91d59feaa3